_projects/cloud-detection.md

```diff
@@ -0,0 +1,43 @@
+---
```

Comment:
This file name contains a dash instead of an underscore (like the instructions above state).

Reply:
good catch.